test/test_contraction_sequence.jl

```diff
@@ -5,6 +5,7 @@ using ITensorNetworks:
 using ITensors: ITensors, contract
 using NamedGraphs.NamedGraphGenerators: named_grid
 using OMEinsumContractionOrders: OMEinsumContractionOrders
+using TensorOperations
```

Comment:
```suggestion
using TensorOperations: TensorOperations
```
To minimize what we bring into the namespace.